draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -548,11 +548,11 @@
 
 				<section anchor="TLVoperationP2P" title="Operation on a point to point interface">
 
-					<t>By sending the LSP Burst Window sub-TLV, a node advertises to its neighbor its ability to receive that many un-acknowledged LSPs from the neighbor, without an intervening delay. This is akin to a receive window or sliding window in flow control. In some implementations, this value should reflect the IS-IS socket buffer size. Special care must be taken to leave space for CSNP and PSNP (SNP) PDUs and IIHs if they share the same input queue. In this case, this document suggests advertising an LSP Burst Window corresponding to half the size of the IS-IS input queue. </t>
+					<t>By sending the Receive Window sub-TLV, a node advertises to its neighbor its ability to receive that many un-acknowledged LSPs from the neighbor, without an intervening delay. This is akin to a receive window or sliding window in flow control. In some implementations, this value should reflect the IS-IS socket buffer size. Special care must be taken to leave space for CSNPs and PSNPs (SNPs) and IIHs if they share the same input queue. In this case, this document suggests advertising an LSP Receive Window corresponding to half the size of the IS-IS input queue. </t>
 
 					<t>By advertising an LSP Transmission Interval sub-TLV, a node advertises its ability to receive LSPs separated by at least the advertised value, outside of LSP bursts.</t>
 
-					<t>The LSP transmitter MUST NOT exceed these parameters. After having sent a full burst of un-acknowledged LSPs, it MUST send the following LSPs with an LSP Transmission Interval between LSP transmissions. For CPU scheduling reasons, this rate may be averaged over a small period, e.g., 10-30ms.</t>
+					<t>The LSP transmitter MUST NOT exceed these parameters. After having sent a full burst of un-acknowledged LSPs, it MUST send the subsequent LSPs with an LSP Transmission Interval between LSP transmissions. For CPU scheduling reasons, this rate MAY be averaged over a small period, e.g., 10-30ms.</t>

```

Comment:
To elaborate on the burst size parameter description, I think it's fine for an implementation to send bursts every time, it allows to reduce the number of timer events.

Also, maybe 
OLD: it MUST send the subsequent LSPs with an LSP Transmission Interval between LSP transmissions
NEW: it MUST send the subsequent LSPs with a minimum of LSP Transmission Interval between LSP transmissions.

Otherwise it sounds like it's mandating the sender to precisely send each LSP, while it's really a lower bound

Reply:
Thank you. Applied.